packages/fxa-payments-server/src/routes/Subscriptions/PaymentUpdateForm.tsx

```diff
@@ -69,17 +65,9 @@ export const PaymentUpdateForm = ({
 
   // clear any error rendered with `ErrorMessage`
   const onChange = useCallback(() => {
-    if (createTokenError.error) {
-      setCreateTokenError({ type: '', error: false });
-    } else if (updatePaymentStatus.error) {
-      resetUpdatePayment();
-    }
-  }, [
-    createTokenError,
-    updatePaymentStatus,
-    setCreateTokenError,
-    resetUpdatePayment,
-  ]);
+    setCreateTokenError({ type: '', error: false });
+    resetUpdatePayment();
```

Comment:
What's the reason behind this change?  It no longer checks for errors at all.

Reply:
[This goes back to a comment in the original PR](https://github.com/mozilla/fxa/pull/3266#discussion_r343873520) - the error checking here is just to decide which kind of error to reset, on the assumption that either reset is expensive.

But a) they're not really expensive and b) the added conditional branches dragged down test coverage, because we didn't have test cases testing all three scenarios. Rather than add new tests, I just took out the conditional.